### PR TITLE
Bugfix/hagberg/log vomit

### PR DESCRIPTION
**Pull request by thagberg**
Re-enable noisy logs, change argument from `quiet` to `noisy` and make it opt-in.

### CODE REVIEW

File: electron/renderer.js [hunk 1]
```diff
@@ -168,21 +167,20 @@ class Renderer {
...
-        // }
+        // Create new bindings for `console` functions
+        // Use default console[name] and also send IPC
+        // log so we can log to stdout
+        function bindConsole() {
+            for (const name in console) {
+                if (typeof console[name] === 'function') {
+                    globalLoggers[name] = console[name];
+                    console[name] = function(...args) {
+                        globalLoggers[name].apply(console, args);
```
Comment (mbittarelli) [inline electron/renderer.js]: you probably want to add my changes in here as well so we don't crash the console if someone logs an object with a circular dependency
something like:
```
                         let depthLimitStr = str;
                         if(typeof depthLimitStr === "object") {
                             depthLimitStr = util.inspect(depthLimitStr, {depth: this.logDepth });
                         }
                         remote.process.stdout.write(depthLimitStr);
```

**Comment by bigtimebuddy** [comment]
Suggest `verbose` instead of `noisy`

**Comment by thagberg** [comment]
@bigtimebuddy I thought about that, but I wasn't sure since it's not actually changing the logging level, just adding a lot of mostly unwanted logs to stdout which would normally only exist in dev tools.

**Comment by mbittarelli** [comment]
something to think about the "opt-in" strategy.. the actual command that jenkins will run will be different than what most people do, so we'll just need to update that as well for all the jobs that run

**Comment by mbittarelli** [comment]
LGTM

**Comment by bigtimebuddy** [comment]
Closing, floss imported to monorepo.

**Comment by francoislaberge** [comment]
Everybody wins! :)

**Pull request closed without merging**